### ISSUE TITLE
docker/build.sh demo_android -it bash fails

### ISSUE THREAD

**Issue by herbakamil** [dmlc/tvm#3958]
https://github.com/dmlc/tvm/blob/9e4f07b4695a8849590cdd46de662e3fa273d59b/docker/Dockerfile.demo_android#L70

Command fails with errors like:
```
CMake Error at cmake/util/FindLLVM.cmake:76 (string):
  string sub-command STRIP requires two arguments.
Call Stack (most recent call first):
  cmake/modules/LLVM.cmake:22 (find_llvm)
  CMakeLists.txt:240 (include)
```
And more generally it's hard to follow the tutorial to optimize for Android.
- On my Ubuntu 19.04 Java 11 is installed by default with which sources are not compatible
- docker/bash.sh tvmai/demo-android fails
- building image fails

So the only way to run the tutorial is to prepare custom docker image where all needed resources will be avilable

**Comment by tqchen** [comment]
@junrushao1994 can you look into this? is it related to your recent change

**Comment by junrushao1994** [comment]
The recent change is simply a single liner “separate_argument(LLVM_CONFIG)”
though...I will take a look!

**Comment by junrushao1994** [comment]
@tqchen @herbakamil I looked into the docker container, but found that there is no llvm-config-6.0 at all...

I can confirm that build succeeded when Dockerfile uses llvm-config-8 instead.

**Comment by tqchen** [comment]
Hmm, perhaps two take aways, we should fix the docker file, and ideally we should have nicer error messages when we cannot find llvm-config. @junrushao1994 can you help on that?
Thanks for looking into the problem!

**Comment by junrushao1994** [comment]
@tqchen 

**Dockerfile.** I wonder if `Dockerfile.demo_android` is included in CI. I was thinking that it shouldn't pass CI anyway bc the build won't succeed without proper `llvm-config`.

**CMake.** It shouldn't be hard to have such error message, i can do this

**Comment by tqchen** [comment]
The CI does not build docker images, we manually build docker images and upload them to the docker hub. This is something that we overlooked, we could try to add a nightly that rebuild docker images if necessary in the future.

**Comment by herbakamil** [comment]
To me as the person who is testing tvm on our model the big surprise was that there was no ready docker image to test Android tuning tutorials. Because tutorials are very nice. Also I had to think a while which dockerfile is appropriate and finally I modified the one for android_ci because this one I guess id the most comprehensive. Anyway I will see if today I manage to run these tutorials.

**Comment by tqchen** [comment]
There is actually a docker image here https://cloud.docker.com/u/tvmai/repository/docker/tvmai/demo-android please let us know if there is any place where we should highlight it in our documents. Contribution to the docs is more than welcomed

**Comment by herbakamil** [comment]
It seems to be not usable:

```
docker pull tvmai/demo-android
```

```
Using default tag: latest
Error response from daemon: manifest for tvmai/demo-android:latest not found: manifest unknown: manifest unknown
```

**Comment by herbakamil** [comment]
Also only 2 pulls might be a proof that something is wrong with this image

**Comment by junrushao1994** [comment]
idk what is going on here. let's reopen this issue? @herbakamil

**Comment by tqchen** [comment]
Thanks for the note, I just updated the image, please try again.
